### PR TITLE
Nwap 3091 camera Position changed  callback

### PR DESCRIPTION
**Pull request by dydkask**

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,838 @@
...
+        <state relative-caret-position="441">
+          <caret line="76" column="45" selection-start-line="76" selection-start-column="45" selection-end-line="76" selection-end-column="45" />
+          <folding>
+            <element signature="e#0#41#0" expanded="true" />
+          </folding>
+        </state>
+      </provider>
+    </entry>
+  </component>
+</project>
```
Comment (felangel) [inline .idea/workspace.xml]: Can we remove this file please?

File: packages/atlas/lib/src/atlas.dart [hunk 2]
```diff
@@ -33,6 +33,11 @@ class Atlas extends StatelessWidget {
   /// which can be used to manipulate the map.
   final ArgumentCallback<AtlasController> onMapCreated;
 
+  /// When the camera position is changed
+  /// Callback Receiving Camera Location
+  /// Recall during map manipulation
```
Comment (felangel) [inline packages/atlas/lib/src/atlas.dart]: Thoughts on using onCameraPositionChanged for consistency?

**Review by felangel — DISMISSED** [review]
See comments

File: .idea/workspace.xml (hunk 1 above)
Comment (dydkask) [reply .idea/workspace.xml]: i will fix

File: packages/atlas/lib/src/atlas.dart (hunk 2 above)
Comment (dydkask) [reply packages/atlas/lib/src/atlas.dart]: ok i will fix that

Comment (felangel) [inline packages/atlas/lib/src/atlas.dart]:
```suggestion
  /// `onCameraPositionChanged ` gets called when the map's camera position is changed.
  /// The updated `CameraPosition ` is passed as an argument.
```

**Review by dakrawczyk — APPROVED** [review]

**Review by tchabot22 — APPROVED** [review]

**Pull request merged**